src/acquisition/covidcast/csv_to_database.py

```diff
@@ -146,9 +151,8 @@ def main(args):
 
   logger = get_structured_logger("csv_ingestion", filename=args.log_file)
   start_time = time.time()
-
   # shortcut escape without hitting db if nothing to do
-  path_details = collect_files(args.data_dir, args.specific_issue_date)
+  path_details = collect_files(args.data_dir, args.specific_issue_date, indicator_name = args.indicator_name)
```

Comment:
```suggestion
  path_details=collect_files(args.data_dir, args.specific_issue_date, indicator_name = args.indicator_name)
```
nit: when specifying keyword arguments, the convention is to omit spaces around the equals